src/mongoose-cleaner.js

```diff
@@ -0,0 +1,36 @@
+import _ from 'lodash';
```

Comment:
I'd recommend not reinventing any wheels, even though they're  easy to build. :) We can spend that time on something else. ;)